src/RecordManager/Base/Record/Lido.php

```diff
@@ -220,27 +220,32 @@ public function toSolrArray(Database $db = null)
     /**
      * Return record title
      *
-     * @param bool     $forFiling            Whether the title is to be used in
+     * @param bool   $forFiling            Whether the title is to be used in
      *                                       filing (e.g. sorting, non-filing
      *                                       characters should be removed)
-     * @param string   $lang                 Language
-     * @param string[] $excludedDescriptions Description types to exclude
+     * @param string $lang                 Language
+     * @param array  $excludedDescriptions Description types to exclude
      *
      * @return string
      */
     public function getTitle(
-        $forFiling = false,
-        $lang = null,
-        $excludedDescriptions = ['provenance']
-    ) {
+        bool $forFiling = false,
```

Comment:
If you add the types here, you'll need to do it for the base class and all other derivatives as well (see the failed tests). Better to do that in a separate refactoring task that does not change functionality.